pkg/controller/global/setup.go

```diff
@@ -3,17 +3,18 @@ package global
 import (
 	"context"
 
-	"github.com/rancher/steve/pkg/server"
-
 	"github.com/rancher/harvester/pkg/config"
 	"github.com/rancher/harvester/pkg/controller/global/settings"
+	"github.com/rancher/harvester/pkg/controller/global/template"
+	"github.com/rancher/steve/pkg/server"
 )
 
 type registerFunc func(context.Context, *config.Scaled, *server.Server) error
 
 var registerFuncs = []registerFunc{
 	createCRDs,
 	settings.Register,
+	template.Register,
```

Comment:
Can we put this to master controller register

Reply:
It will run initData after the server started, need crds applied to the cluster first before we call templateClient to create template, so put initData in the global level after createCRDs. If we put then in the master, for the server newly deployed harvester, crds may not ready when the master started.